src/main/java/org/broadinstitute/consent/http/resources/DarCollectionResource.java

```diff
@@ -87,4 +115,19 @@ private void validateUserIsCreator(User user, DarCollection collection) {
       throw new NotFoundException();
     }
   }
+
+  private Map<String, String> defineAcceptableSortFields() {
+    return Map.of(
+      "projectTitle", "projectTitle",
+      "researcher", "researcher",
+      "darCode", "dar_code",
+      "institution", "institution_name"
+    );
+  }
+
+  private void checkIfSortFieldAndSortOrderIsPresent(String sortField, String sortOrder) {
+    if (Objects.isNull(sortOrder) || Objects.isNull(sortOrder)) {
+      throw new BadRequestException("sortOrder and sortField are required");
+    }
```

Comment:
See above, I think we should choose a sensible default and let the client make different choices if preferred.